.github/workflows/validate.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v2
         with:
-          go-version: "${{ env.GO_VERSION }}"
+          go-version: 1.17.x # TODO switch to "${{ env.GO_VERSION }}"
```

Comment:
Looks like go 1.18 may be supported now (although it looks like they will skip some linters, based on some tickets linked to that one); not sure if we're running all of those though; https://github.com/golangci/golangci-lint/pull/2438

Reply:
Let me try again )

Reply:
Yeah, no promises; just noticed the linked ticket was closed, so perhaps we could have everything done in one go 👍 🤞